js/app.js

```diff
@@ -1,126 +1,84 @@
-// Enemies our player must avoid
-var Enemy = function(x,y) {
-    // Variables applied to each of our instances go here,
-    // we've provided one for you to get started
-    // The image/sprite for our enemies, this uses
-    // a helper we've provided to easily load images
-    //Agregar Comentario
-    this.sprite = "images/enemy-bug.png";
-    this.x = x;
-    this.y = y;
-    this.dt = 10.0;
-};
 
-// Update the enemy's position, required method for game
-// Parameter: dt, a time delta between ticks
-Enemy.prototype.update = function(dt) {
-    // You should multiply any movement by the dt parameter
-    // which will ensure the game runs at the same speed for
-    // all computers.
-
-    if(this.x >= 500){
-        this.x = -100;
-        this.dt = Math.floor((Math.random() * 10) + 1);
-        //this.dt = Math.random()*100;
-        //allEnemies.push(enemy);
+class Enemy {
```

Comment:
because this is part of the ES2015 version of the js standard, so most browser(older versions) don't have support for the feature since is new.  [ES2015](https://themeteorchef.com/blog/what-is-es2015/) take a look to this link.